### PR TITLE
docs(ReplyMessageOptions,MessageEditOptions): replaced `embed` with `embeds`

### PR DESCRIPTION
**Pull request by MidSpike**
**Please describe the changes this PR makes and why it should be merged:**
- Removed `ReplyMessageOptions#embed` to match typings

**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

**Comment by MidSpike** [comment]
I meant to push [those edits](https://github.com/discordjs/discord.js/pull/5860/commits/d36d5da1fb1358162115b43b1b8d0cee72290583) to a separate branch -_-

**Comment by kyranet** [comment]
I can't suggest changes in deleted lines, so...

## Short story

You need to copy those two lines into `ReplyMessageOptions` (so the end result is that you replace `embed` with `embeds`):

https://github.com/discordjs/discord.js/blob/f7eeccba4b7015496df811f10cc2da2b0fab0630/src/structures/interfaces/TextBasedChannel.js#L73-L74

---

## Long story

`ReplyMessageOptions` is defined as `MessageOptions` without the `reply` options in the typings:

https://github.com/discordjs/discord.js/blob/f7eeccba4b7015496df811f10cc2da2b0fab0630/typings/index.d.ts#L3765-L3767

`MessageOptions` being:

https://github.com/discordjs/discord.js/blob/f7eeccba4b7015496df811f10cc2da2b0fab0630/typings/index.d.ts#L3473-L3484

However, in the JSDocs, this differs a little, see the code you modified:

https://github.com/MidSpike/discord.js/blob/d36d5da1fb1358162115b43b1b8d0cee72290583/src/structures/Message.js#L646-L651

You can see it extends `BaseMessageOptions` (which is created as we can't `Omit<>` in JSDocs), we can see below that `embeds` is not a valid property:

https://github.com/discordjs/discord.js/blob/f7eeccba4b7015496df811f10cc2da2b0fab0630/src/structures/interfaces/TextBasedChannel.js#L54-L68

However, it is in `MessageOptions`:

https://github.com/discordjs/discord.js/blob/f7eeccba4b7015496df811f10cc2da2b0fab0630/src/structures/interfaces/TextBasedChannel.js#L70-L76

File: src/structures/Message.js [hunk 1]
```diff
@@ -646,8 +646,8 @@ class Message extends Base {
   /**
    * Options provided when sending a message as an inline reply.
    * @typedef {BaseMessageOptions} ReplyMessageOptions
-   * @property {MessageEmbed|Object} [embed] An embed for the message
-   * (see [here](https://discord.com/developers/docs/resources/channel#embed-object) for more details)
+   * @property {MessageEmbed[]|Object[]} [embeds] The embeds for the message 
+   * (see [here](https://discord.com/developers/docs/resources/channel#embed-object) for more details)
```
Comment (kyranet) [inline src/structures/Message.js:650]:
```suggestion
   * @property {MessageEmbed[]|Object[]} [embeds] The embeds for the message
   * (see [here](https://discord.com/developers/docs/resources/channel#embed-object) for more details)
```

**Review by kyranet — CHANGES_REQUESTED** [review]
A review a day makes ESLint go away

**Review by kyranet — APPROVED** [review]

**Comment by PapiOphidian** [comment]
No other Option Types seem to have the embed field if this PR goes through. BaseMessageOptions can now include the embeds field and other Option Types can have their declaration of embeds removed if they extend BaseMessageOptions.

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**